rhizome/common/lib/util.rb

```diff
@@ -65,3 +66,9 @@ def safe_write_to_file(filename, content)
     File.rename(temp_filename, filename)
   end
 end
+
+def change_os_user(user)
```

Comment:
I kinda wonder if this is a good idea in this situation relative to `sudo -u` or `systemd-run --uid`. I have written such programs before, but only when transitioning in the middle of a program is necessary, whereas here, it seems like it runs in the beginning only?

Reply:
Removed this for now. We can add it back if we need it.